packages/create-redwood-app/templates/ts/web/tsconfig.json

```diff
@@ -3,9 +3,9 @@
     "noEmit": true,
     "allowJs": true,
     "esModuleInterop": true,
-    "target": "esnext",
-    "module": "esnext",
-    "moduleResolution": "node",
+    "target": "ES2022",
+    "module": "ES2022",
```

Comment:
Sorry if I'm beating on a dead horse here, but if I understand TypeScript's docs correctly, what we have here now is an invalid combination of settings.

> `--moduleResolution bundler` must be paired with `--module esnext` or `--module preserve`.

https://www.typescriptlang.org/docs/handbook/modules/reference.html#bundler

Reply:
It does look like I have broken the rules here. If you want to revert then we can do so.

I find it interesting that unlike other combinations of settings that do, this combination does not throw an error when you use it. So using terms like "must" and "enforced" on their docs is a bit confusing to me. 

`ES2022` is also identical to `ESNext` at the current time. I also don't understand why this constraint would exist anyway. It's not my job to question these rules though haha.

I was just keen to move to a set version rather than `ESNext` and for a modern resolution algorithm to use. Do you have any suggestions to fix or change this?

Reply:
Follow-up PR here: https://github.com/redwoodjs/redwood/pull/11368